tgstation.dme

```diff
@@ -6,6 +6,66 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"

```

Comment:
u didn't do the thing 

https://tgstation13.org/phpBB/viewtopic.php?f=5&t=321&p=106881